assets/css/_comments.scss

```diff
@@ -23,10 +23,20 @@
 }
 
 .comments-list {
-  border-top: 5px solid $light-gray;
+  border-top: $base-border;
   list-style-type: none;
   margin-top: $base-spacing;
   padding-top: $base-spacing;
+  position: relative;
+
+  &::before {
+    content: "Comments";
```

Comment:
Just curious, why do you prefer this to an `<h2>` (or whatever) in the DOM? I imagine that would be more accessible.

Reply:
You're right. I more or less mocked this up in browser and left it. I'll transition to more accessible content.